src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -2140,7 +2136,13 @@ private bool CanUseConnectEx(EndPoint remoteEP)
 
         internal IAsyncResult UnsafeBeginConnect(EndPoint remoteEP, AsyncCallback? callback, object? state, bool flowContext = false)
         {
-            if (SocketsTelemetry.Log.IsEnabled()) SocketsTelemetry.Log.ConnectStart(remoteEP);
+            if (SocketsTelemetry.Log.IsEnabled())
+            {
+                SocketsTelemetry.Log.ConnectStart(remoteEP);
```

Comment:
I can't find the matching `AfterConnect` in the catch block within `BeginConnectEx`. Am I missing something?

Reply:
Fixed.